frontend/templates/product/index.tsx

```diff
@@ -23,6 +25,16 @@ export const ProductTemplate: NextPageWithLayout<ProductTemplateProps> = () => {
    const { product } = useProductTemplateProps();
    const [selectedVariant, setSelectedVariantId] = useSelectedVariant(product);
 
+   React.useEffect(() => {
+      trackMatomoEcommerceView({
+         productSku: selectedVariant.sku ?? selectedVariant.id,
+         productName:
+            selectedVariant.internalDisplayName?.value ?? product.title,
+         price: selectedVariant.price,
+      });
+      // eslint-disable-next-line react-hooks/exhaustive-deps
+   }, []);
```

Comment:
Why doesn't this depend on selectedVariant?

Reply:
As I understand it, in ifixit we just track the default selected option, instead of recording a product view every time the user changes the selected variant in the UI.
https://github.com/iFixit/ifixit/blob/5b354e81eace15e414f45b26595003a6c7ec6462/carpenter-frontend/Cart/product_view.js#L919

The empty array only tracks this product once. If we depend on selectedVariant, then we will track the product every time the user switches the variant.

Reply:
@jarstelfox does that sound right?

Reply:
@masonmcelvain that does sound correct.

This comes down to a business decision. Do we want the view product to be the first view product xor do we want the viewed product plus any viewed variants?

Reply:
Let's keep the status quo for V1 of the Next.js product page, and let folks that use Minerva decide if they want to change this later on.